gcovr/tests/test_gcovr.py

```diff
@@ -127,7 +127,7 @@ def scrub_xml(contents):
 
 def scrub_html(contents):
     contents = RE_HTML_ATTRS.sub('\\1=""', contents)
-    contents = RE_HTML_HEADER_DATE.sub("\\1>0000-00-00 00:00:00<\\2", contents)
+    contents = RE_HTML_HEADER_DATE.sub("\\1>0000-00-00 00:00:00<\\3", contents)
```

Comment:
Offset isn't captured anymore:
```suggestion
    contents = RE_HTML_HEADER_DATE.sub("\\1>0000-00-00 00:00:00<\\2", contents)
```